charts/timesketch/values-production.yaml

```diff
@@ -0,0 +1,280 @@
+## @section Redis Configuration Parameters
+## IMPORTANT: Redis is deployed with Auth enabled by default
+## To see a full list of available values, run helm show values charts/redis*
+##
+redis:
+  ## @param redis.enabled Enables the Redis deployment
+  ##
+  enabled: true
+  ## @param redis.sentinel.enabled Enables Redis Sentinel on Redis pods
+  ## IMPORTANT: This has not been tested for Timesketch so would leave this disabled
+  ##
+  sentinel:
+    enabled: false
+  ## Master Redis Service configuration
+  ##
+  master:
+    ## @param redis.master.count Number of Redis master instances to deploy (experimental, requires additional configuration)
+    ##
+    count: 1
+    ## Redis master persistence configuration
+    ##
+    persistence:
+      ## @param redis.master.persistence.size Redis master Persistent Volume size
+      ##
+      size: 8Gi
+    ## Redis master resource requests and limits
+    ## @param redis.master.resources.requests.cpu Requested cpu for the Redis master containers
+    ## @param redis.master.resources.requests.memory Requested memory for the Redis master containers 
+    ## @param redis.master.resources.limits.cpu Resource cpu limits for the Redis master containers
+    ## @param redis.master.resources.limits.memory Resource memory limits for the Redis master containers
+    ##
+    resources:
+      requests:
+        cpu: 4000m
+        memory: 8Gi
+      limits:
+        cpu: 8000m
+        memory: 16Gi
+  ## Redis replicas configuration parameters
+  ##
+  replica:
+    ## @param redis.replica.replicaCount Number of Redis replicas to deploy
+    ##
+    replicaCount: 3
+    ## Redis replicas persistence configuration
+    ##
+    persistence:
+      ## @param redis.replica.persistence.size Redis replica Persistent Volume size
+      ##
+      size: 8Gi
+    ## Redis Replica resource requests and limits
+    ## @param redis.replica.resources.requests.cpu Requested cpu for the Redis replica containers
+    ## @param redis.replica.resources.requests.memory Requested memory for the Redis replica containers 
+    ## @param redis.replica.resources.limits.cpu Resource cpu limits for the Redis replica containers
+    ## @param redis.replica.resources.limits.memory Resource memory limits for the Redis replica containers
+    ##
+    resources:
+      requests:
+        cpu: 4000m
+        memory: 8Gi
+      limits:
+        cpu: 8000m
+        memory: 16Gi 
+## @section Postgresql Configuration Parameters
+## IMPORTANT: Postgresql is deployed with Auth enabled by default
+## To see a full list of available values, run helm show values charts/postgresql*
+##
+postgresql:
+  ## @param postgresql.enabled Enables the Postgresql deployment
+  ##
+  enabled: true
+  ## PostgreSQL Primary configuration parameters
+  ##
+  primary:
+    ## PostgreSQL Primary persistence configuration
+    ##
+    persistence:
+      ## @param postgresql.primary.persistence.size PostgreSQL Persistent Volume size
+      ##
+      size: 8Gi
```

Comment:
This is a little low for a production deployment - make it something like 500Gi?

Reply:
Agreed. Updated all prod persistent volumes to 500Gi minimum